Wrappers/Python/conda-recipe/meta.yaml

```diff
@@ -35,7 +35,7 @@ requirements:
     - numpy
     - vtk
     - pyside2
-    - eqt=0.7.1
```

Comment:
I think this should be explicitly request eqt version 1 as the change is backwards incompatible

Reply:
ok, changed